lms/djangoapps/commerce/signals.py

```diff
@@ -104,11 +104,27 @@ def refund_seat(course_enrollment, request_user):
             course_key_str,
             refund_ids,
         )
-        try:
-            send_refund_notification(course_enrollment, refund_ids)
-        except:  # pylint: disable=bare-except
-            # don't break, just log a warning
-            log.warning("Could not send email notification for refund.", exc_info=True)
+
+        # XCOM-371: this is a temporary measure to suppress refund-related email
+        # notifications to students and support@) for free enrollments.  This
+        # condition should be removed when the CourseEnrollment.refundable() logic
+        # is updated to be more correct, or when we implement better handling (and
+        # notifications) in Otto for handling reversal of $0 transactions.
+        if course_enrollment.mode != 'verified':
+            # 'verified' is the only enrollment mode that should presently
+            # result in opening a refund request.
+            log.info(
+                "Skipping refund email notification for non-verified mode for user [%s], course [%s], mode: [%s]",
+                course_enrollment.user,
```

Comment:
i'll fix that (it was already logging this in one other place in the same module, which i'll also fix).